generics/services/survey.js

```diff
@@ -262,9 +262,135 @@ var getObservationDetail = function ( solutionId, token ) {
 
 }
 
+/**
+  * Get survey documents.
+  * @function
+  * @name getStartedSurveys
+  * @param {String} token - logged in user token.
+  * @param {String} programId - program Id
+  * @returns {Promise} returns a promise.
+*/
+
+const getStartedSurveys = function(token, programId) {
+    let url = 
+    process.env.ML_SURVEY_SERVICE_URL + 
+    constants.endpoints.GET_IMPORTED_SURVEY
+
+    if( programId !== "" ) {
+        url += "/" + programId;
+    }
+    
+    return new Promise(async (resolve, reject) => {
+        try {
+
+            function assessmentCallback(err, data) {
+
+                let result = {
+                    success : true,
+                    message: "",
+                    status:""
+                };
+
+                if (err) {
+                    result.success = false;
+                } else {
+                    
+                    let response = JSON.parse(data.body);
+                    if( response.status === httpStatusCode['ok'].status ) {
+                        result["result"] = response.result;
+                    } else {
+                        result.success = false;
+                    }
+
+                    result.message = response.message;
+                    result.status = response.status;
+                }
+
+                return resolve(result);
+            }
+
+            const options = {
+                headers : {
+                    "content-type": "application/json",
+                    "x-authenticated-user-token" : token
+                }
+            };
+
+            request.get(url,options,assessmentCallback)
+
+        } catch (error) {
+            return reject(error);
+        }
+    })
+}
+
+
+/**
+  * Get observation documents.
+  * @function
+  * @name getStartedObservations
+  * @param {String} token - logged in user token.
+  * @param {String} programId - program Id
+  * @returns {Promise} returns a promise.
+*/
+const getStartedObservations = function(token, programId) {
```

Comment:
@ankitpws Rename this to userObservations

Reply:
Done